tests/unit/TodoList2.spec.ts

```diff
@@ -57,6 +61,95 @@ describe('TodoList - common expansion panel header tests', () => {
     wrapper.destroy()
   })
 
+  it('display affiliation invitation todo (request access, open)', async () => {
+    rootStore.tasks = []
+    businessStore.businessInfo = Object.assign(businessStore.businessInfo, { identifier: 'testIdentifier' })
+
+    const affiliationInvitationsForEntityUrl = 'entity/testIdentifier/affiliations/invitations'
+
+    const sandbox = sinon.createSandbox()
+    const axiosGet = sandbox.stub(axios, 'get')
+
+    // feature flag override
+    sandbox.stub(utils, 'GetFeatureFlag')
+      .withArgs('enable-affiliation-invitation-request-access')
+      .returns(true)
+
+    axiosGet.withArgs(affiliationInvitationsForEntityUrl).returns(new Promise(
```

Comment:
Dunno where my previous comment went, but...

I recommend stubbing the auth service call directly instead of the axios call. (This test suite is not responsible for testing the services.)

Reply:
That is fair, replaced the stub from axios to auth-service.